diff --git a/node_modules/astro-icon/lib/Icon.astro b/node_modules/astro-icon/lib/Icon.astro
index cfd0d9c..ba9c670 100644
--- a/node_modules/astro-icon/lib/Icon.astro
+++ b/node_modules/astro-icon/lib/Icon.astro
@@ -1,6 +1,6 @@
 ---
 import load, { fallback, normalizeProps } from './utils.ts';
-export { Props } from './Props.ts';
+export type { Props } from './Props.ts';
 
 let { name, pack, title, optimize = true, class: className, ...inputProps } = Astro.props;
 let props = {};
@@ -14,14 +14,10 @@ try {
     innerHTML = svg.innerHTML;
     props = svg.props;
 } catch (e) {
-    if (import.meta.env.MODE === 'production') {
-        throw new Error(`[astro-icon] Unable to load icon "${name}"!
-${e}`)
-    }
     innerHTML = fallback.innerHTML;
     props = { ...fallback.props, ...normalizeProps(inputProps) };
     title = `Failed to load "${name}"!`
-    console.error(e);
+    console.warn(`[astro-icon] Unable to load icon "${name}": ${e && e.message ? e.message : e}`);
 }
 ---
 
diff --git a/node_modules/astro-icon/lib/Sprite.astro b/node_modules/astro-icon/lib/Sprite.astro
index f64c5d6..305697f 100644
--- a/node_modules/astro-icon/lib/Sprite.astro
+++ b/node_modules/astro-icon/lib/Sprite.astro
@@ -2,7 +2,7 @@
 import { SPRITESHEET_NAMESPACE } from './constants';
 import { normalizeProps } from './utils.ts';
 import { trackSprite } from './context.ts';
-export { Props } from './Props.ts';
+export type { Props } from './Props.ts';
 
 let { name, pack, title, class: className, x, y, ...inputProps } = Astro.props;
 const props = normalizeProps(inputProps);
diff --git a/node_modules/astro-icon/lib/utils.ts b/node_modules/astro-icon/lib/utils.ts
index 120df7b..7eb3cbd 100644
--- a/node_modules/astro-icon/lib/utils.ts
+++ b/node_modules/astro-icon/lib/utils.ts
@@ -38,7 +38,7 @@ function optimizeSvg(
       "minifyStyles",
       "convertStyleToAttrs",
       {
-        name: "cleanupIDs",
+        name: "cleanupIds",
         params: { prefix: `${SPRITESHEET_NAMESPACE}:${name}` },
       },
       "removeRasterImages",
